project/SparkBuild.scala

```diff
@@ -741,7 +741,8 @@ object Unidoc {
     javacOptions in (JavaUnidoc, unidoc) := Seq(
       "-windowtitle", "Spark " + version.value.replaceAll("-SNAPSHOT", "") + " JavaDoc",
       "-public",
-      "-noqualifier", "java.lang"
+      "-noqualifier", "java.lang",
+      "-tag", """note:a:Note\:"""
```

Comment:
@srowen, this was added to deal with `@note` annotation in Java documentation. It seems `@note` for scaladoc not for javadoc. So, I had to manually deal with this.